### DISCUSSION TITLE
Where's the source code for tree-sitter.github.io?

### DISCUSSION THREAD

**Discussion by giampaolo** [tree-sitter/tree-sitter#2967] [Q&A] [answered]
Hello!
I am not a user of tree-sitter, but I bumped into https://tree-sitter.github.io/tree-sitter/ by accident and I like the website style **a lot**, as it seems perfect to document a project and its APIs. Is it open source? If yes, where is the code hosted?

Thanks a lot

**Comment by dundargoc** [comment]
https://github.com/tree-sitter/tree-sitter.github.io

**Reply by amaanq** [reply, marked as answer]
partially, there's also https://github.com/tree-sitter/tree-sitter/blob/master/docs/
